examples/real/qrdqn_monolith_discrete_real.py

```diff
@@ -0,0 +1,241 @@
+import argparse
+import os
+import pprint
+
+import numpy as np
+import torch
+from examples import DQN,QRDQN
+from examples import wrap_offworld_real
+from torch.utils.tensorboard import SummaryWriter
+
+from tianshou.data import Collector, VectorReplayBuffer
+from tianshou.env import ShmemVectorEnv, DummyVectorEnv
+from tianshou.policy import QRDQNPolicy
+from tianshou.trainer import offpolicy_trainer
+from tianshou.utils import TensorboardLogger
+
+import gym
+import offworld_gym
+from offworld_gym.envs.common.channels import Channels
+from offworld_gym.envs.common.enums import AlgorithmMode, LearningType
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning) # to surpress the warning when running real env
+
+def get_args():
```

Comment:
Please updated README (of the "examples" section) by adding a default run command that reproduces the results you've obtained

Also -- please update readme to only list what exists in this branch and fixing broken links